src/components/Header/Header.js

```diff
@@ -2,13 +2,17 @@ import React from 'react';
 import styled from 'styled-components';
 
 import { useSelector } from 'react-redux'
+import Sidebar from '../Sidebar/Sidebar'
 
 const Header = ({title}) => {
-
+    const [sidebarToggle, setSidebarToggle] = React.useState('false')
     return(
-        <Headerpiece>
-            <TextDiv>{title}</TextDiv>
-        </Headerpiece>
+        <>
+            {sidebarToggle === 'true' ? <Sidebar sidebarToggle={sidebarToggle} setSidebarToggle={setSidebarToggle}/> : <Sidebar setSidebarToggle={setSidebarToggle} sidebarToggle={sidebarToggle}/>}
+            <Headerpiece onClick = {() => setSidebarToggle('true')}>
```

Comment:
Why do you use string instead of boolean for `sideBarToggle` ? should be `sideBarToggle === true ?` or event better `sideBarToggle ?` and you can initialize it using `React.useState(false)` better than using string as this is prone to typo